Meetings/2023-03-07.md

```diff
@@ -2,19 +2,33 @@
 - Zoom Link: https://cuboulder.zoom.us/j/346594091
 
 ### Attending
-- ?
-
+- Ross Beyer
+- Andrew Annex
+- Ross Beyer
```

Comment:
I was only there for like 10 minutes, how did I attend twice?

Reply:
Also, if you remove my name, then it was only me and then Andrew?  Surely there were more people there--you were taking notes!